### PR TITLE
In Ontime 17 separator between Issue ID and Subject was changed from NewLine to Tab

### PR DESCRIPTION
**Pull request by PaulStupka**
OnTimeValueResolver was changed appropriately.

### CODE REVIEW

File: OnTimeIssueFolderMakerPlant/OnTime/OnTimeValueResolver.cs [hunk 1]
```diff
@@ -18,13 +18,25 @@ public class OnTimeValueResolver
...
     {
-      return this.ResolveNewOntime(headerValue) ?? this.ResolveOldOntime(headerValue);
+      return this.ResolveOntime17(headerValue) ?? this.ResolveNewOntime(headerValue) ?? this.ResolveOldOntime(headerValue);
     }
 
     #endregion
 
     #region Methods
 
+    private Tuple<int, string> ResolveOntime17(string bufferValue)
```
Comment (zvirja) [inline OnTimeIssueFolderMakerPlant/OnTime/OnTimeValueResolver.cs]: This method looks like a copy of the `ResolveNewOntime` one with the difference in the delimiter. Wouldn't it be more appropriately to just introduce one more parameter for the delimiter and re-use the `ResolveNewOntime` method for both `NewLine` and `\t` delimiters?

**Review by zvirja — CHANGES_REQUESTED** [review]
Thank you for the PR - I shed a tear from the fact that somebody uses it 😢 

I left a small comment - could you please improve the code a bit? :)

**Comment by PaulStupka** [comment]
Well... I saw it coming :)
Thank you!

**Review by zvirja — APPROVED** [review]

**Comment by zvirja** [comment]
Merged, thanks for your contribution 👍

**Pull request merged**